src/tablecloth/.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+jobs:
+  bucklescript-build-and-test:
+    docker:
+      - image: circleci/node:11
+
+    environment:
+      CI: true
+      NODE_ENV: test
+
+    working_directory: ~/repo
+    steps:
+      - checkout:
+          path: ~/repo
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ .Branch }}-{{ checksum "bs/package-lock.json" }}
+            - v1-dependencies-{{ .Branch }}-
+            - v1-dependencies-
+      - run: make deps-bs
+      - run: make build-bs
+      - run: make test-bs
+      - run: make doc-bs
+      - save_cache:
+          key: v1-dependencies-{{ .Branch }}-{{ checksum "bs/package-lock.json" }}
+          paths:
+            - ~/repo/bs/node_modules
+
+  native-build-and-test:
+    docker:
+      - image: ocaml/opam2
+    environment:
+      CI: true
+    working_directory: ~/repo
+    steps:
+      - checkout:
+          path: ~/repo
+      - restore_cache:
+          keys:
+            - v2-native-dependencies-{{ .Branch }}-{{ checksum "tablecloth-native.opam" }}
+            - v2-native-dependencies-{{ .Branch }}-
+            - v2-native-dependencies-
+      # m4 is a system dependency required by conf-m4 -> ocamlfind -> fmt -> alcotest
+      - run: sudo apt-get install -y m4
+      - run: make deps-native
+      - run: make build-native
+      - run: make test-native
+      - run: make doc-native
+      - save_cache:
+          key: v2-native-dependencies-{{ .Branch }}-{{ checksum "tablecloth-native.opam" }}
+          paths:
+            - ~/.opam
+
+  source-code-formatting:
+    docker:
+      - image: ocaml/opam2
+    environment:
+      CI: true
+    working_directory: ~/repo
+    steps:
+      - checkout:
+          path: ~/repo
+      # m4 is a system dependency required by conf-m4 -> ocamlfind -> fmt -> alcotest
+      # fd (aka fdfind) is needed to support formatting. It will respect the 's .gitignore
+      - run: sudo apt-get install -y m4 fd-find
+      - run: opam update && opam install 'ocamlformat=0.10' -y
```

Comment:
Should be updated to a recent ocamlformat (or at least leave a TODO) here.

Reply:
Tablecloth has been removed